apps/models/app_types/shiny.py

```diff
@@ -23,16 +23,21 @@ class ShinyInstance(BaseAppInstance, SocialMixin, LogsEnabledMixin):
         ("public", "Public"),
         ("link", "Link"),
     )
+
+    volume = models.ForeignKey(
+        "VolumeInstance", blank=True, null=True, related_name="%(class)s", on_delete=models.CASCADE
+    )
     access = models.CharField(max_length=20, default="project", choices=ACCESS_TYPES)
     port = models.IntegerField(default=3838)
     image = models.CharField(max_length=255)
+    path = models.CharField(max_length=255, default="/")
     proxy = models.BooleanField(default=True)
     container_waittime = models.IntegerField(default=20000)
     heartbeat_timeout = models.IntegerField(default=60000)
     heartbeat_rate = models.IntegerField(default=10000)
-    minimum_seats_available = models.IntegerField(default=2)
-    seats_per_container = models.IntegerField(default=3)
-    allow_container_reuse = models.BooleanField(default=True)
+    minimum_seats_available = models.IntegerField(default=1)
+    seats_per_container = models.IntegerField(default=1)
+    allow_container_reuse = models.BooleanField(default=False)
```

Comment:
Information on this is possible to find via PRs and jira, but for convenience, maybe we'll add a comment with a short description what do these settings do?

And I think that's just me not knowing how the whole apps deployment pipeline work, but how does it play with the [values file in the shiny proxy chart](https://github.com/ScilifelabDataCentre/serve-charts/pull/49/files#diff-11cbabbb8a92fa9b735ed4f34d7ae2575c2ab1f62b5d6659f323eb069d530f8c)?

Reply:
Good point. I will add some clarifying comments. These values override the chart default values.